### PR TITLE
Night theme for UI set

### PR DESCRIPTION
**Pull request by DmitrySvetlichny**

### CODE REVIEW

File: Sources/Controllers/TargetMenu/Search/OAHistoryTableViewController.mm [hunk 1]
```diff
@@ -494,8 +494,8 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             {
                 NSArray *nib = [[NSBundle mainBundle] loadNibNamed:[OAFilledButtonCell getCellIdentifier] owner:self options:nil];
                 cell = (OAFilledButtonCell *)[nib objectAtIndex:0];
-                cell.button.backgroundColor = [UIColorFromRGB(color_primary_purple) colorWithAlphaComponent:0.1];
-                [cell.button setTitleColor:[UIColorFromRGB(color_primary_purple) colorWithAlphaComponent:1.0] forState:UIControlStateHighlighted];
+                cell.button.backgroundColor = [UIColor.buttonBgColorPrimary colorWithAlphaComponent:0.1];
+                [cell.button setTitleColor:[UIColor.buttonTextColorPrimary colorWithAlphaComponent:1.0] forState:UIControlStateHighlighted];
```
Comment (tigrim) [inline Sources/Controllers/TargetMenu/Search/OAHistoryTableViewController.mm:498]: only UIColor.buttonTextColorPrimary

File: Sources/Controllers/OsmEditing/OAOsmEditingViewController.mm [hunk 2]
```diff
@@ -140,20 +140,29 @@ - (void) viewWillAppear:(BOOL)animated
...
+    self.navigationController.navigationBar.standardAppearance = blurAppearance;
     self.navigationController.navigationBar.scrollEdgeAppearance = appearance;
-    self.navigationController.navigationBar.tintColor = UIColor.whiteColor;
+    self.navigationController.navigationBar.tintColor = UIColor.navBarTextColorPrimary;
     self.navigationController.navigationBar.prefersLargeTitles = NO;
     
     _backButton = [[UIBarButtonItem alloc] initWithImage:[UIImage templateImageNamed:@"ic_navbar_chevron"] style:UIBarButtonItemStylePlain target:self action:@selector(onBackPressed)];
     [self.navigationController.navigationBar.topItem setLeftBarButtonItem:_backButton animated:YES];
     
+    self.segmentContainerView.backgroundColor = [self.navigationController.navigationBar.scrollEdgeAppearance.backgroundColor colorWithAlphaComponent:1.];
```
[the remark this answers is not in the document]
Comment (DmitrySvetlichny) [reply Sources/Controllers/OsmEditing/OAOsmEditingViewController.mm:164]: Its correct

File: Sources/Controllers/TargetMenu/GPX/TrackMenu/OAAddWaypointViewController.xib [hunk 3]
```diff
@@ -100,13 +101,15 @@
...
-                            <color key="backgroundColor" red="0.3411764705882353" green="0.078431372549019607" blue="0.80000000000000004" alpha="1" colorSpace="calibratedRGB"/>
+                            <color key="backgroundColor" name="buttonBgColorPrimary"/>
                             <constraints>
                                 <constraint firstAttribute="height" constant="42" id="hcX-bH-hT3"/>
                             </constraints>
                             <fontDescription key="fontDescription" type="system" weight="semibold" pointSize="15"/>
-                            <color key="tintColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                            <state key="normal" title="Add"/>
+                            <color key="tintColor" name="buttonTextColorPrimary"/>
+                            <state key="normal" title="Add">
```
[the remark this answers is not in the document]
Comment (DmitrySvetlichny) [reply Sources/Controllers/TargetMenu/GPX/TrackMenu/OAAddWaypointViewController.xib:110]: Yes

File: Sources/Controllers/TargetMenu/GPX/TrackMenu/OATrackMenuHeaderView.mm [hunk 4]
```diff
@@ -147,18 +147,18 @@ - (void)updateHeader:(BOOL)currentTrack
         if (!self.directionContainerView.hidden)
         {
             self.directionIconView.image = [UIImage templateImageNamed:@"ic_small_direction"];
-            self.directionIconView.tintColor = UIColorFromRGB(color_primary_purple);
-            self.directionTextView.textColor = UIColorFromRGB(color_primary_purple);
+            self.directionIconView.tintColor = UIColor.iconColorActive;
```
[the remark this answers is not in the document]
Comment (DmitrySvetlichny) [reply Sources/Controllers/TargetMenu/GPX/TrackMenu/OATrackMenuHeaderView.mm:150]: The colors are the same, just iconColorActive for icons and textColorActive for text, just to match the name

**Review by tigrim — APPROVED** [review]

**Pull request merged**